rend3/src/graph/graph.rs

```diff
@@ -539,13 +540,13 @@ impl<'node> RenderGraph<'node> {
     }
 
     #[allow(clippy::too_many_arguments)]
-    fn create_rpass_from_desc<'rpass>(
+    fn create_rpass_from_desc<'rpass, A>(
         desc: &RenderPassTargets,
         encoder: &'rpass mut CommandEncoder,
         node_idx: usize,
         pass_end_idx: usize,
         resource_spans: &'rpass FastHashMap<GraphResource, ResourceSpan>,
-        active_views: &'rpass FastHashMap<TextureRegion, TextureView>,
+        active_views: &'rpass FastHashMap<TextureRegion, (TextureView, A)>,
```

Comment:
The Arc<Texture> was unused, so I added a generic parameter here that's unused just to document that the function doesn't really carea bout this. Hopefully that's not too weird. Let me know if you can think of a better approach!